pkg/archive/changes.go

```diff
@@ -386,7 +386,21 @@ func ExportChanges(dir string, changes []Change) (Archive, error) {
 		// during e.g. a diff operation the container can continue
 		// mutating the filesystem and we can see transient errors
 		// from this
+	changesLoop:
```

Comment:
though in golang, these named loops allow for much cleaner `break`ing of the outside loop.